service/src/main/java/org/kiwiproject/champagne/dao/HostDao.java

```diff
@@ -22,6 +22,9 @@ public interface HostDao {
     @GetGeneratedKeys
     long insertHost(@BindBean Host host, @Bind("tagCsv") String tagCsv);
 
+    @SqlUpdate("update hosts set hostname = :hostname, tags = :tags where id = :id")
+    int updateHost(@Bind("hostname") String hostname, @Bind("tags") String tagCsv, @Bind("id") long id);
```

Comment:
Longer term (or maybe now), would it be better to use an array type in the database? Pros include more semantically correct and maybe easier to query, cons might include less portability across DBs. And more I have not thought about.

Reply:
I'll create a ticket to look into switching it out.